frontend/src/components/common/SideBar/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button } from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
+import { FC, useState } from "react";
+import {
+  SideBar as SideBarWrapper,
+  SideBarClosedButtonWrapper,
+  SideBarOpenButtonWrapper,
+  SideBarPositioner,
+} from "src/components/common/SideBar/style";
+
+export interface Props {
+  label: string;
+  isOpen?: boolean;
+  width?: number;
+}
+
+const SideBar: FC<Props> = ({
+  children: content,
+  label,
+  isOpen = false,
+  width = 240, // default side bar width 240px
+}) => {
+  const [isExpanded, setIsExpanded] = useState<boolean>(isOpen);
```

Comment:
TS can infer this to be `boolean`, since `isOpen` is boolean
```suggestion
  const [isExpanded, setIsExpanded] = useState(isOpen);
```